ui/app/AppLayouts/Profile/stores/AdvancedStore.qml

```diff
@@ -119,6 +121,10 @@ QtObject {
           // toggle history archive support
           advancedModule.toggleCommunityHistoryArchiveSupport()
         }
+        else if (feature === experimentalFeatures.wakuV2StoreEnabled) {
+          // toggle history archive support
+          advancedModule.toggleWakuV2Store()
```

Comment:
Looks like most lines are using 4 spaces for code inside of `if` block. Maybe you can follow that rule too ? :)